x-pack/plugins/monitoring/server/lib/metrics/classes.js

```diff
@@ -495,13 +491,13 @@ export class LogstashClusterMetric extends Metric {
     super({
       ...opts,
       app: 'logstash',
-      timestampField: 'logstash_stats.timestamp'
+      ...LogstashClusterMetric.getMetricFields()
     });
   }
 
-  // helper method
   static getMetricFields() {
     return {
+      uuidField: 'cluster_uuid',
```

Comment:
What was this addition necessary? Asking because I don't see this line being removed from the object being passed to `super` a few lines above here, which is what a lot of the other changes in this PR are doing.

Reply:
The intent for `*ClusterMetric` classes is to aggregate metrics on the level of clusters, instead of per-instance. So `uuidField` should always be `cluster_uuid` for these. It only has an effect if something consumes the `getMetricFields` method, and there are many places in the code where instead of consuming the method, a module would just hardcode the field into a query, and I think that's ok. Having all the cluster metric classes set `cluster_uuid` as the `uuidField` just helps make sure a hard-to-detect issue wouldn't happen.